info/Community/irc.md

```diff
@@ -43,7 +43,7 @@ You could use something else, if you’d like (anything will work as long as it
 
 ### Channels
 
-Here are some of the popular channels on the server:
+Here are some of the popular channels on the server (these are also the channels bridged to Discord):
```

Comment:
Maybe move this to the **Discord** heading?

Reply:
That would be up to @newbold if he wants to have a bigger Discord section. I was just going with what was already there.